src/pages/search.js

```diff
@@ -33,7 +33,74 @@ class Search extends React.Component {
   script.setAttribute("defer", true)
 
   document.body.appendChild(script)
+*/
+    const script2 = document.createElement("script") // Loads the Addsearch JS blob from them
+    script2.setAttribute(
+      "src",
+      `https://cdn.jsdelivr.net/npm/addsearch-js-client@0.8/dist/addsearch-js-client.min.js`
+    )
+    //script2.setAttribute("defer", true)
+    document.body.appendChild(script2)
+
+
+    const script3 = document.createElement("script") // Loads the Addsearch JS blob from them
+    script3.setAttribute(
+      "src",
+      `https://cdn.jsdelivr.net/npm/addsearch-search-ui@0.7/dist/addsearch-search-ui.min.js`
+    )
+   // script3.setAttribute("defer", true)
+    script3.onload = () => this.addSearchStuff();
+   document.body.appendChild(script3)
+
+
+    const link = document.createElement("script") // Loads the Addsearch JS blob from them
+    link.setAttribute(
+      "href",
+      `https://cdn.jsdelivr.net/npm/addsearch-search-ui@0.7/dist/addsearch-search-ui.min.css`
+    )
+    link.setAttribute("rel", 'stylesheet')
+
+    //alert('mount up');
+    document.body.appendChild(link)
+
+
+
+
+
+
 }
+ addSearchStuff() {
+  //alert('add it');
+   var client = new AddSearchClient('a7b957b7a8f57f4cc544c54f289611c6');
+
+
+   var searchui_conf = {
+     //searchResultsPageUrl: 'search'
+     searchResultsPageUrl: 'search'
+   }
+   // Search UI instance
+   var searchui = new AddSearchUI(client, searchui_conf);
+
+   // Add components
+   searchui.searchField({
+     containerId: 'searchfield',
```

Comment:
I think this code is meant to add a form that is different from the form in the header. But looking at https://pr-8665-documentation.appa.pantheon.site/search/, I only see the form in the header